health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -167,39 +167,65 @@ export const UICustomizations = {
+        const onActionSelect = (key, row) => {
+          switch (key) {
+            case "START":
+              window.history.pushState(
+                {
+                  microplanId: row?.id,
+                  campaignId: row?.campaignId,
+                },
+                "",
+                `/${window.contextPath}/employee/microplan/select-activity?microplanId=${row?.id}&campaignId=${row?.campaignId}`
+              );
+              const navEvent = new PopStateEvent("popstate");
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)